Adding deprecation note

matzew opened the pull request:
@sebastienblanc I know you wrote it, but I dont think it does make sense to continue this.

For the new UPS, based on Swarm modules, [see](https://github.com/matzew/aerogear-unifiedpush-server/tree/Updates_dev/swarms), I am only adding those libs, that make sense

matzew committed:
Adding deprecation note

camilamacedo86 commented:
This project is not so longer working with the CI. Please, folks, could you give a hand here for we discovery if it is or not deprecated?

camilamacedo86 committed:
Change in order to keep the same standard of the other projects

## Deprecation Notice
This repository has been deprecated and is not being maintained. If you have any questions, please get in touch with the collaborators.
